src/reference/informative/informC10_interpretation_of_map_variables5.rst

```diff
@@ -0,0 +1,247 @@
+.. _informC10_interpretation_of_map_variables5:
+
+.. container:: toggle
+
+  .. container:: header
+
+    See more
+
+  .. container:: infospec
+
+    .. container:: heading3
+
+      Understanding interfaces
+
+    There are two hurdles to get past before variables can be mapped to one another.
+    The first is the relative position within the encapsulation hierarchy of the :code:`variable` elements' parent :code:`component` elements (as discussed in :ref:`Interpretation of encapsulation<specC_interpretation_of_encapsulation>`, and the second is the value of the :code:`interface_type` attribute given to each :code:`variable` itself.
+    This means that even if two components are sufficiently close relatives that they *can* be connected, the modeller still has control at the individual variable level as to which of those relatives can have access.
+
+    The :code:`interface_type` attribute value choices are: 
+    - :code:`public` (used by child components to reference their parent, or between siblings),
+    - :code:`private` (used by parent components to reference their children), 
+    - :code:`public_and_private` (used by all near relatives), or
```

Comment:
"used if both a public and a private interface is required"

Reply:
Fixed in 9625b88